### PR TITLE
Create Data Storage (View Patient Demographics)

### PR DESCRIPTION
**Pull request by Suracki**
- added docker-compose.yml to set up MySQL database
- created SpringBoot project to interface with data storage
- created basic model/repository implementation

Completed steps to create MySQL database docker for the application to connect to, and completed the initial steps to create an app with the ability to interface with this database, including setting up the domain/model class for the Patient entity

https://trello.com/c/v4voHBKp/1-view-patient-demographics

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,33 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
```
Comment (fpatelm) [inline .gitignore]: remove .mvn and sub folders from git

File: mvnw [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+# ----------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
```
Comment (fpatelm) [inline mvnw]: this file should not be tracked on git

File: mvnw.cmd [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+@REM distributed with this work for additional information
+@REM regarding copyright ownership.  The ASF licenses this file
+@REM to you under the Apache License, Version 2.0 (the
+@REM "License"); you may not use this file except in compliance
+@REM with the License.  You may obtain a copy of the License at
+@REM
+@REM    https://www.apache.org/licenses/LICENSE-2.0
+@REM
+@REM Unless required by applicable law or agreed to in writing,
+@REM software distributed under the License is distributed on an
```
Comment (fpatelm) [inline mvnw.cmd]: this file should not be tracked on git

**Review by fpatelm — CHANGES_REQUESTED** [review]
Remove those tagged files form git tracking system

File: .gitignore (hunk 1 above)
Comment (Suracki) [reply .gitignore]: gitignore updated to ignore maven files/folder

File: mvnw (hunk 2 above)
Comment (Suracki) [reply mvnw]: removed

File: mvnw.cmd (hunk 3 above)
Comment (Suracki) [reply mvnw.cmd]: removed

**Review by fpatelm — APPROVED** [review]
LGTM

**Pull request merged**